setup.py

```diff
@@ -27,23 +31,27 @@
         "License :: OSI Approved :: MIT License",
         "Natural Language :: English",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 2.6",
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.2",
         "Programming Language :: Python :: 3.3",
         "Programming Language :: Python :: 3.4",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
         "Topic :: Scientific/Engineering",
         "Topic :: System :: Filesystems",
         "Topic :: Scientific/Engineering :: Information Analysis",
-    ),
+    ],
     keywords="science geophysics io",
     packages=("lasio",),
     install_requires=("numpy",),
-    extras_require={"all": ("pandas", "cchardet", "openpyxl", "argparse")},
-    tests_require=("pytest>=3.6", "pytest-cov", "coverage", "codecov", "pathlib"),
+    extras_require={
+        "all": EXTRA_REQS,
+        "las_test_reqs": ( EXTRA_REQS, TEST_REQS)
```

Comment:
Can we rename this to "test" instead of "las_test_reqs"?

Reply:
Commit 4679de4 renames "las_test_reqs" to "test"

DC